etc/schema.json

```diff
@@ -1101,7 +1101,7 @@
                 "authorities": {
                     "$ref": "#/$defs/dns.authorities"
                 },
-                "query": {
+                "queries": {
                     "type": "array",
```

Comment:
Did you run a check if the schema is complete ?
cf 9332bc2c45f1e1e08f2261857d5804f41ce7e639

Reply:
It passes existing tests. Do we have another check to run?

Reply:
No tests, but as you reread the code, maybe you found more that are missing and not existing in tests...

Reply:
Looks like `SOA` is missing from code review

Reply:
> Looks like `SOA` is missing from code review

We don't have any tests that log it outside of authorities.  And I know there's been some question whether we should have it in the schema if we don't have a test that exercises it.  Thoughts?

Reply:
Last time, I ended up creating a command for sshfp by running a dig command...

Reply:
Ok, I ended up with some when I added logging all arrays for requests.  A zone update request will log them.